galleon-pack/galleon-shared/src/main/resources/layers/standalone/micrometer/layer-spec.xml

```diff
@@ -8,6 +8,8 @@
         <prop name="org.wildfly.rule.add-on-depends-on" value="only:cdi"/>
         <prop name="org.wildfly.rule.add-on" value="observability,micrometer"/>
```

Comment:
@jasondlee , so this layer is also an add-on? Is there a use case to explicitly enable this layer, even if the deployment doesn't use the micrometer API?

Reply:
@jfdenise I think so. An admin may want server metrics exported via Micrometer even if no deployments are explicitly using Micrometer.

Reply:
@jasondlee , Thank-you!